packages/brisa/src/cli/build-standalone/index.ts

```diff
@@ -79,7 +79,7 @@ async function compileStandaloneServerComponents(
     : brisaServerDeps;
 
   return Bun.build({
-    // TODO: adapt to Bun > 1.2 (for now this is to force the old behavior)
+    // @ts-ignore -- TODO: adapt to Bun > 1.2 (for now this is to force the old behavior)
```

Comment:
why this `@ts-ignore` is needed? Did you update the Brisa dependencies to use `@types/bun` 1.2?

Reply:
It was updated before I arrived. Probably you used force to push before? Or no check?

Reply:
Nope, I always check tsc, and the `throw` property is on Bun 1.2

Reply:
https://github.com/brisa-build/brisa/blob/fe791c4f8707ec88b42af14f7ffcfad7ab0c7ec8/packages/brisa/package.json#L140

Reply:
Something is off on your editor then, we can check together later on :)